admin/server/middleware/browserify.js

```diff
@@ -35,7 +36,16 @@ module.exports = function (file, name) {
 	var logName = typeof file === 'string' ? file.replace(/^\.\//, '') : name;
 	var fileName = logName;
 	if (fileName.substr(-3) !== '.js') fileName += '.js';
+
+	var writeToDisk = keystone.get('write keystone files to disk');
+	var staticPath = keystone.get('static') || path.join(keystone.get('module root'), 'public');
```

Comment:
why isn't there a default for the static folder? can we get away with using one? or should we default to storing some static files inside the keystone repo (gitignored) if the user doesn't declare a static folder?